data/human/ships.txt

```diff
@@ -48,16 +48,18 @@ ship "Aerie"
 		"X3700 Ion Thruster"
 		"X3200 Ion Steering"
 		"Hyperdrive"
-		
-	engine -10 91
-	engine 10 91
-	gun -13 -79 "Sidewinder Missile Launcher"
-	gun 13 -79 "Sidewinder Missile Launcher"
-	turret 0 0 "Heavy Anti-Missile Turret"
-	turret -17 14 "Heavy Laser Turret"
-	turret 17 14 "Heavy Laser Turret"
-	bay "Fighter" -42 -2
-	bay "Fighter" 42 2
+
```

Comment:
Please refrain from eating the whitespace.
```suggestion
		
```

Reply:
The extra engines was simply an attempt to keep design elements from the previous iteration, (if those 6 cylinders were even engine visuals or something else entirely), and aren't super necessary to this new model if you suggest as much. It was more of a "they could fit here" rather than, "they _should_ fit here".

Should there be possibilities of a marauder variant in the future though (not on my to-do list, just a mention), I will keep them in the very back of my head.

Sorry about the whitespace again.